CMakeLists.txt

```diff
@@ -101,7 +101,7 @@ if(PHOEBUS_ENABLE_CUDA)
   set(Kokkos_ENABLE_CUDA_LAMBDA ON CACHE BOOL "" FORCE)
   # Get around a bug with atomics on Cray
   # This option will eventually be deprecated, so care required.
-  set(Kokkos_ENABLE_IMPL_DESUL_ATOMICS OFF CACHE BOOL "" FORCE)
+  # set(Kokkos_ENABLE_IMPL_DESUL_ATOMICS OFF CACHE BOOL "" FORCE)
```

Comment:
Not sure how this will pan out. This line was required by Chicoma because of a Cray bug. But the version of Kokkos Parthenon is pinned to deprecated this flag and it causes Kokkos to fail to compile.